quarry/data/packets.csv

```diff
@@ -80,7 +80,8 @@ minecraft_version,protocol_version,protocol_mode,packet_direction,packet_name
 1.12.1,338,play,downstream,advancements
 1.12.1,338,play,downstream,entity_effect
 1.12.1,338,play,upstream,teleport_confirm
-1.12.1,338,play,upstream,prepare_crafting_grid
+1.12.1,338,play,upstream,craft_recipe_request
+1.12.1,338,play,downstream,craft_recipe_response
 1.12.1,338,play,upstream,tab_complete

```

Comment:
Packet order - you can see on http://wiki.vg/Protocol that the mention the "Packet ID" and we keep the packets in the needed order in the data instead of clogging it up with packet IDs.